Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.9",
+	"GoVersion": "go1.10",
```

Comment:
This should not be changed.

Reply:
Sorry, forgot that I switched to golang 1.10.

Reply:
Done.